src/TSID/src/JointRegularizationTask.cpp

```diff
@@ -0,0 +1,160 @@
+/**
+ * @file JointRegularizationTask.cpp
+ * @authors Giulio Romualdi
+ * @copyright 2021 Istituto Italiano di Tecnologia (IIT). This software may be modified and
+ * distributed under the terms of the GNU Lesser General Public License v2.1 or any later version.
+ */
+
+#include <BipedalLocomotion/TSID/JointRegularizationTask.h>
+
+#include <iDynTree/Core/EigenHelpers.h>
+
+using namespace BipedalLocomotion::ParametersHandler;
+using namespace BipedalLocomotion::TSID;
+
+bool JointRegularizationTask::initialize(
+    std::weak_ptr<ParametersHandler::IParametersHandler> paramHandler,
+    const System::VariablesHandler& variablesHandler)
+{
+    constexpr std::string_view errorPrefix = "[JointRegularizationTask::initialize] ";
+
+    constexpr std::string_view descriptionPrefix = "JointRegularizationTask Optimal Control "
+                                                   "Element";
```

Comment:
I am not sure this adds any detail in the error messages. Also, it is no more an ``Optimal Control Element``.

Reply:
Remove here: https://github.com/dic-iit/bipedal-locomotion-framework/pull/170/commits/273400091eb76b53c7e8ebb8d687bb0ff25945ee